docs/usage_guide.rst

```diff
@@ -1015,17 +1025,22 @@ The liquidity pool support allows premium users to see their balances, the per p
 Liabilities
 =============
 
-In the liabilities section you can find information on your Aave Borrowing, Compound Borrow, and MakerDAO Vaults.
+In the liabilities section you can find information on your Aave Borrowing, Compound Borrow, Liquity troves and MakerDAO Vaults.
 These collateralized loans can be autodetected from your ethereum accounts and information about each one of them is displayed.
 However you can manage the different modules and addresses that are queried to make the retrieval faster.
 For more information you can check :ref:`customize-module-settings`.
 
-As a normal non-premium user you can see all your vaults, and for each one inspect the locked collateral, collateralization, debt generated and the liquidation price.
+As a normal non-premium user you can see all your vaults/troves, and for each one inspect the locked collateral, collateralization, debt generated and the liquidation price.
```

Comment:
What about premium users? Any screenshots of that? We would need it. It's something we need to show off to entice people to buy.